src/sage/misc/latex_macros.py

```diff
@@ -175,7 +175,12 @@ def convert_latex_macro_to_mathjax(macro):
 latex_macros = [r"\newcommand{\SL}{\mathrm{SL}}",
                 r"\newcommand{\PSL}{\mathrm{PSL}}",
                 r"\newcommand{\lcm}{\mathop{\operatorname{lcm}}}",
-                r"\newcommand{\dist}{\mathrm{dist}}"]
+                r"\newcommand{\dist}{\mathrm{dist}}",
+                r"\newcommand{\im}{\mathop{\operatorname{im}}}",
+                r"\newcommand{\rank}{\mathop{\operatorname{rank}}}",
+                r"\newcommand{\PP}{\mathbf{P}}",
+                r"\newcommand{\OO}{\mathcal{O}}",
```

Comment:
are we sure that we want to create the macro `\OO`?

Reply:
I also have qualm about it.  I need it since I use it extensively for docstrings in `src/sage/schemes/projective/cohomology.py`. On the other hand,  I feel a bit guilty for increasing the size of the list `latex_macros` since this will slow down documentation build...

Reply:
But I have no idea how to circumvent this dilema...

Reply:
Sheaves are written in calligraphic font, if this is what you asked. I am following the convention.

Reply:
Macaulay2's convention seen in 

https://macaulay2.com/doc/Macaulay2-1.21/share/doc/Macaulay2/Macaulay2Doc/html/_sheaf_lp__Ring_rp.html

also affected me.

Reply:
Apologies for not asking a more specific question. Mainly I asked because the double-letter command suggests boldface/double-stroke to me, and this is how all of our existing double-letter commands expand. For the calligraphic O, I would expect to see many different conventions for latex macros including `\calO` or similar. (Note in IPython, one types `\scrO` + TAB to get this symbol.) The M2 reference may be convincing enough though.

Reply:
Yes, `\calO` may be alternative, but I prefer `\OO` to `\calO` because of the M2 reference.
 
For the general calligraphic O, I think the vanilla `\mathcal{O}` is not much worse than `\calO`.

Reply:
On the other hand, `\OO` is a latex macro only useful for algebraic geometry while the list `latex_macros` is supposed to keep macros in general use. I think in the long run, we should devise some mechanism to define latex macros for local use within the sage library... Right now I have no idea what the mechanism should be like...

Reply:
However, when someone claims `\OO` for boldfaced O, then we should reconsider this issue seriously :-)

Reply:
OK. To keep the good taste, I'll not mention the terrible things I've seen that were done using \mathbb{O}.

Reply:
I cannot assert that I never saw boldfaced O in papers...